backend/src/database/migrations/20221116100713-update-stop-jobs.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createFunction(
+      'stop_jobs',
+      [{ type: 'integer', name: 'selected_campaign_id' }],
+      'void',
+      'plpgsql',
+      `
+UPDATE job_queue SET status = 'STOPPED', updated_at = clock_timestamp(), visible_at = clock_timestamp() WHERE campaign_id = selected_campaign_id AND status <> 'LOGGED';
```

Comment:
why do we need to update `visible_at` in `stop_job` again?

Reply:
oh yes good point, removed.